x-pack/plugins/fleet/public/applications/integrations/sections/epm/screens/detail/custom_languages_overview.tsx

```diff
@@ -21,9 +21,7 @@ export const CustomLanguagesOverview = () => {
   const { pkgkey } = useParams<CustomLanguageClientsParams>();
   const { getPath } = useLink();
 
-  const Component = getCustomIntegrationsStart().languageClientsUiComponents.get(
-    `language_client.${pkgkey}`
-  );
+  const Component = getCustomIntegrationsStart().languageClientsUiComponents[pkgkey];
```

Comment:
I wasn't sure about this, but the prefix on the package key here seemed unneeded. Was there a good reason for this? Maybe to prevent collisions with integrations?

Reply:
it seems not needed to me too

Reply:
Yes you can remove it